rpcs3/Emu/system_config.h

```diff
@@ -134,7 +133,11 @@ struct cfg_root : cfg::node
 		cfg::_bool disable_vulkan_mem_allocator{ this, "Disable Vulkan Memory Allocator", false };
 		cfg::_bool full_rgb_range_output{ this, "Use full RGB output range", true, true }; // Video out dynamic range
 		cfg::_bool strict_texture_flushing{ this, "Strict Texture Flushing", false };
+#ifdef __APPLE__
+		cfg::_bool disable_native_float16{ this, "Disable native float16 support", true };
```

Comment:
Ditto

Reply:
Because Megamouse preferred everything in a single line.

Reply:
Less lines of code, also "making this file easier to follow."